infra/main.tf

```diff
@@ -49,4 +49,11 @@ resource "aws_lambda_function" "fastapi" {
 
   handler = "main.handler"
   runtime = "python3.8"
+
+  environment {
+    variables = {
+      table_name = "${var.DYNAMODB_TABLE_NAME}"
```

Comment:
`table_name = var.DYNAMODB_TABLE_NAME`

${} only if formatting it as a string with other values

Reply:
actually it should be

`DYNAMODB_TABLE_NAME = var.dynamodb_table_name`